opensaml/3.3.1.wso2v3/pom.xml

```diff
@@ -0,0 +1,1151 @@
+<!--
+  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change year to 2021 and check if there is a new license file.

Reply:
Fixed.